src/converters/degiroConverterV3.ts

```diff
@@ -62,90 +69,83 @@ export class DeGiroConverterV3 extends AbstractConverter {
         activities: []
       };
 
+      const isinHardcodeMapping = new Map<string, string>();
+      if (process.env.DEGIRO_V3_ISIN_HARDCODE_MAPPING) {
+        /* Sometimes there are cases where getting security gets a wrong result.
+         * For instance dividends for IE00B02KXK85 (ISHARES CHINA LARGE CAP UCITS ETF USD)
+         * which are in USD are detected as Invesco CurrencyShares Canadian Dollar Trust (FXC).
+         * So, if an imported file has first dividend payment the rest of the transactions
+         * are registered under wrong security. If first transaction is buy/sell, then it's all good
+         * since buy/sell txs are in EUR.
+         *
+         * This hack allows to buypass all internal heuristic and hardcode correct results.
+         * Format: IE00B02KXK85=FXAC.AS,foo=bar
```

Comment:
I have seen a similar thing with the isin of VWRL resulting in VWRL.L instead of VWRL.AS (see comment on line 92 on main).

I think you’re on the right path to a solution, but I think it can be improved upon. I’d then rather use a simple text file with mappings which you as a user can provide.

I have created #149 for this to make this separate of this change. Could you revert any change regarding this mapping override, so the this PR is moreso a better fix for the transaction matching?

Reply:
Do you have ready code? If not, I can write it since I need it. Do you have any particular preferences implementation wise?

Reply:
No I have no code yet, but your suggestion gave me an idea on how to do it.

You can give it a go, but preferably separate from, and after this change has been merged

Reply:
> No I have no code yet, but your suggestion gave me an idea on how to do it.
> 
> You can give it a go, but preferably separate from, and after this change has been merged

so, just a plain file with `foo=bar` pairs on each line?

Reply:
That was basically what I was going for. Then read it when present, and also provide a file volume mount in Docker.

But as stated, I’d prefer it separate from this PR